publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponse.java

```diff
@@ -30,18 +30,23 @@
 
   private static final String TIME_TO_NEXT_CALL = "timeToNextCall";
   private static final String SLOTS = "slots";
+  private static final String CONSENT_GIVEN = "consentGiven";
```

Comment:
What do you think about transforming this class into an Java `@AutoValue` class or a Kotlin `data` class ?

Deserialization is isolating potential error per slots, but except the prefetch (which might be removed), all bid response contains only a single slot.

Reply:
I think this is a good idea, but I don't think we should do this in this PR. It should be done separately.